addons/mail/static/src/components/message/message.js

```diff
@@ -448,14 +449,20 @@ class Message extends Component {
             }
             return;
         }
-        this.state.isClicked = !this.state.isClicked;
+        if (!isEventHandled(ev, 'Message.ClickAuthorAvatarHandled')
+            && !isEventHandled(ev, 'Message.ClickAuthorNameHandled')
+            && !isEventHandled(ev, 'Message.ClickFailureHandled')
+           ) {
```

Comment:
It's a bit more clear with our usual syntax
```suggestion
        if (
            !isEventHandled(ev, 'Message.ClickAuthorAvatarHandled') &&
            !isEventHandled(ev, 'Message.ClickAuthorNameHandled') &&
            !isEventHandled(ev, 'Message.ClickFailureHandled')
        ) {
```